### ISSUE TITLE
Insufficient contrast ratios in the ui

### ISSUE THREAD

**Issue by jennschiffer** [tastejs/todomvc-app-css#30]
todomvc apps are very hard for me to look at because of how light and small all the text is. i ran a a11y test on [the backbone example](http://todomvc.com/examples/backbone/) using [tota11y](http://khan.github.io/tota11y/) and it showed that there were numerous insufficient contrast ratios (there's also no label for the input, but that's an issue for another todomvc repo i imagine).

i didn't see any info on how to contribute to this repo, so i'm leaving this issue instead of working on a pr, but i would be happy to help with making some css changes so that the ui is easy for more people to see!

thanks for your work on todomvc and also your time reading this <3

**Comment by samccone** [comment]
Hey Jenn, thanks for opening the issue. This seems like a great opportunity to make some revisions to the styling to make sure that the application passes basic contrast rules.

The process for updating styles is somewhat... non-obvious but essentially it goes like this

1. update the CSS in this repo
1.b bump the package.json and publish a new version
2. (happy to do this) Update all of the projects in todomvc to refer to the latest a11y fixed css version and commit the changes back into the repo.
3. \o/

Let me know how I can help!

Thanks again for filing the issue.
